buildenv/Build_Instructions_V8.md

```diff
@@ -384,6 +396,15 @@ bash configure --disable-ccache \
                --with-freetype-lib=/cygdrive/c/temp/freetype/lib64
 ```
 
+2) Win 32-bit
+```
+bash configure --disable-ccache \
+			   --with-boot-jdk=/cygdrive/c/temp/jdk7 \
```

Comment:
Compared to 64-bit, these lines are all indented differently. Perhaps because of different tab width. Use spaces.

Reply:
Replaced with spaces.